x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_by_rule_panel/alerts_by_rule.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { EuiBasicTableColumn } from '@elastic/eui';
+import { EuiInMemoryTable, EuiSpacer, EuiText } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import type { SortOrder } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { ALERT_RULE_NAME } from '@kbn/rule-data-utils';
+import { TableId } from '@kbn/securitysolution-data-table';
+import type { AlertsByRuleData } from './types';
+import { FormattedCount } from '../../../../common/components/formatted_number';
+import { DefaultDraggable } from '../../../../common/components/draggables';
+import { ALERTS_HEADERS_RULE_NAME } from '../../alerts_table/translations';
+import { COUNT_TABLE_TITLE } from '../alerts_count_panel/translations';
+
+const Wrapper = styled.div`
+  margin-top: -${({ theme }) => theme.eui.euiSizeM};
+`;
+const TableWrapper = styled.div`
+  height: 178px;
+`;
+
+export interface AlertsByRuleProps {
+  data: AlertsByRuleData[];
+  isLoading: boolean;
+}
+
+const COLUMNS: Array<EuiBasicTableColumn<AlertsByRuleData>> = [
+  {
+    field: 'rule',
+    name: ALERTS_HEADERS_RULE_NAME,
+    'data-test-subj': 'detectionsTable-rule',
```

Comment:
not important, but if this is new, I would consider making this `data-test-subj` and the one right below a bit more specific, maybe mention it's for alerts kips?